### DISCUSSION TITLE
Criação da migration

### DISCUSSION THREAD

**Discussion by joshuajka** [balta-io/forum#450] [Tira Dúvidas] [answered]
Balta, na crição da migration tive que alterar o campo InvariantGlobalization para "false" no arquivo da solução .csproj , do contrário com true ele não me permitia criar a migration, sabe me explicar o porquê ?

![duvidas balta](https://github.com/balta-io/forum/assets/55669412/d51b0274-af39-411b-b54e-aef890330424)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @joshuajka, como vai?

Acredito que durante a migração possa ter ocorrido algum problema com datas ou outra informação que pode ser globalizada por exemplo.

Foi a única coisa que me veio a cabeça no momento!
